mm2src/lp_ordermatch.rs

```diff
@@ -2558,17 +2511,7 @@ pub async fn lp_ordermatch_loop(ctx: MmArc) {
                             spawn({
                                 let ctx = ctx.clone();
                                 async move {
-                                    if let Ok(Some((base, rel))) =
-                                        find_pair(&ctx, &maker_order.base, &maker_order.rel).await
-                                    {
-                                        maker_order_created_p2p_notify(
-                                            ctx,
-                                            &maker_order,
-                                            base.coin_protocol_info(),
-                                            rel.coin_protocol_info(),
-                                        )
-                                        .await;
-                                    }
+                                    maker_order_created_p2p_notify(ctx, &maker_order).await;
```

Comment:
@shamardy Could you please clarify, what was the reason to remove `if let Ok(Some((base, rel))) = find_pair(&ctx, &maker_order.base, &maker_order.rel).await`? It opens a possibility to have an unmatchable maker order in the orderbook unless the coins are activated.

Reply:
I removed it because I added it for maker order protocol info. on this PR https://github.com/KomodoPlatform/atomicDEX-API/pull/1017 as it wasn't there before. But will add it again to this PR https://github.com/KomodoPlatform/atomicDEX-API/pull/1054 as we agreed since it appears to have another purpose.